File: src/JavaScript/Joint Jog Jam/JointJogJam.js
const readline = require('readline');
const rl = readline.createInterface(process.stdin);

rl.on('line', line => {
  const [x1, y1, x2, y2, k1, o1, k2, o2] = line.split(' ').map(Number);
  dist1 = Math.sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2);
  dist2 = Math.sqrt((k1 - k2) ** 2 + (o1 - o2) ** 2);
  console.log(dist1 > dist2 ? dist1 : dist2);
  rl.close();
})

File: src/Java/Joint Jog Jam/JointJogJam.java
import java.util.Scanner;

public class JointJogJam {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    double[] distance = new double[2];
    for (int i = 0; i < 2; i++) {
      int k1 = sc.nextInt();
      int k2 = sc.nextInt();
      int o1 = sc.nextInt();
      int o2 = sc.nextInt();
      distance[i] = Math.sqrt(Math.pow(k1 - o1, 2) + Math.pow(k2 - o2, 2));
    }
    System.out.println(distance[0] > distance[1] ? distance[0] : distance[1]);
    sc.close();
  }
}